### DISCUSSION TITLE
How to Build C++ Examples in Reaktoro?

### DISCUSSION THREAD

**Discussion by glanzkaiser** [reaktoro/reaktoro#291] [Q&A]
Hi all,

I already finish installing reaktoro with Conda from this link:

https://reaktoro.org/installation/installation-using-conda.html

I am able to run Python examples from the github repository too.

Now, I want to run C++ example and can't do `cmake ` at the github root directory.

I try to run `conda devenv`, but obtain `no command conda devenv`

I try to run `cmake -S . -B build`, not working either.

Is there another way to run C++ examples without cmake?

**Comment by allanleal** [comment]
Hi @glanzkaiser , did you install conda-devenv? This is instructed in this cmake installation guide:
https://reaktoro.org/installation/installation-using-cmake.html

![image](https://user-images.githubusercontent.com/5825588/224090604-4964059e-66a8-46cb-9c1c-41f13e8a2b0d.png)

> Is there another way to run C++ examples without cmake?

Currently, Reaktoro can only be built using cmake, which is the current standard for configuring/programming the build of C++ software.

**Comment by glanzkaiser** [comment]
Hi,

After install `conda-deveenv`,  I already on this stage:

**cmake -S . -B build**

then I get error when try to build reaktoro with cmake:


```
-- Setting build type to Release as none was specified.
CMake Error at cmake/ReaktoroFindDeps.cmake:22 (find_package):
  Could not find a configuration file for package "autodiff" that is
  compatible with requested version "0.6.12".

  The following configuration files were considered but not accepted:

    /home/browni/.julia/conda/3/envs/reaktoro/lib/cmake/autodiff/autodiffConfig.cmake, version: 1.0.0

Call Stack (most recent call first):
  cmake/ReaktoroFindDeps.cmake:32 (ReaktoroFindPackage)
  CMakeLists.txt:93 (include)

```
